javascript-source/systems/auctionSystem.js

```diff
@@ -19,14 +19,30 @@
     var auction = {
             increments: 0,
             minimum: 0,
-            topUser: 0,
+            topUser: 'None',
             topPoints: 0,
             timer: 0,
-            status: false
+            isActive: false,
+            isEnding: false,
+            usePoints: true
         },
         a,
-        b;
+        b,
+        c,
+        //endTimer,
+        extTime = $.getSetIniDbNumber('auctionSettings', 'extTime', 15),
```

Comment:
Codacy has a fix for the issue: [Replace `········extTime·=·$.getSetIniDbNumber('auctionSettings',·'extTime'` with `····extTime·=·$.getSetIniDbNumber("auctionSettings",·"extTime"`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
    extTime = $.getSetIniDbNumber("auctionSettings", "extTime", 15),
```